### ISSUE TITLE
Update existing slim maps in Period 1

### ISSUE THREAD

**Issue by karlcz** [nih-cfde/cfde-deriva#335]
- [x] Create new PR with revised term/map files
- [x] Merge and test (Karl)

**Comment by karlcz** [comment]
We heard on today's call that these updates are not planned for the Q4 release, but may come later in May/June timeframe...

**Comment by karlcz** [comment]
A branch "slim-maps" has been created with a new taxonomy-slim table, but is still waiting on the corresponding taxonomy term list update.

**Comment by karlcz** [comment]
The assay type slim map was also updated in time for the QA period, so both taxonomy and assay type slim maps are changing in the next release.

**Comment by karlcz** [comment]
This was release already.
